crates/ruff_server/src/server/api/requests/code_action_resolve.rs

```diff
@@ -75,6 +80,43 @@ pub(super) fn resolve_edit_for_fix_all(
     Ok(action)
 }
 
+pub(super) fn resolve_edit_for_organize_imports(
+    mut action: types::CodeAction,
+    document: &crate::edit::Document,
+    url: &types::Url,
+    mut linter_settings: ruff_linter::settings::LinterSettings,
+    encoding: PositionEncoding,
+) -> crate::Result<types::CodeAction> {
+    linter_settings.rules = [
+        ruff_linter::registry::Rule::from_code("I001").unwrap(),
```

Comment:
I think you can reference the rule directly
```suggestion
        Rule::UnusedImport
```